src/main/java/com/alkemy/ong/service/ContactService.java

```diff
@@ -0,0 +1,25 @@
+package com.alkemy.ong.service;
+
+import com.alkemy.ong.dto.ContactDto;
+import com.alkemy.ong.entity.ContactEntity;
+import com.alkemy.ong.mapper.ContactMapper;
+import com.alkemy.ong.repository.ContactRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ContactService {
+    @Autowired
+    ContactMapper contactMapper;
+
+    @Autowired
+    ContactRepository contactRepository;
```

Comment:
This @Autowired bean means we are no implementing the mapstruct dependencia. My suggestion please let me write the mapper so we are consistent in the implementation of the mapstruct